kano_updater/scenarios.py

```diff
@@ -934,6 +935,21 @@ def beta_370_to_beta_380(self, dummy_progress):
                 {'kw_app': 'tux-typing', 'disk_req': 26},
                 {'kw_app': 'libreoffice', 'disk_req': 385},
             ]
+            phases = [
+                Phase(
+                    app['kw_app'],
+                    _("Installing {} from the App Store")
+                    .format(app['kw_app'])
+                )
```

Comment:
Might want to give some weighting to each app ([`weight=<val>`](https://github.com/KanoComputing/kano-updater/blob/034115a98ac8a7933d8f252cdac2c2cead05ad41/kano_updater/progress.py#L44)) to give the progress a hint about how much time each phase is likely to take so it can report the progress more accurately - `libreoffice` will take a lot longer to install than `gmail`. You could well use the `disk_req` as this value.